setup.py

```diff
@@ -45,42 +45,18 @@
 ###############################################################################
 # Key Values that Change Each Release
 ###############################################################################
-SETUP_REQUIREMENTS = {
-    "numpy": "1.17",  # released July 2019
-    "scipy": "1.3",  # released May 2019
+# These are strictly installation requirements. Builds requirements are
+# managed in pyproject.toml
+INSTALL_REQUIREMENTS = {
+    "numpy": "1.18",  # released December 2019
```

Comment:
@bashtage We were having issues using statsmodels with Python 3.9 on a Red Hat cluster due to the install requirement on numpy 1.17 (see https://github.com/numpy/numpy/issues/14147).    Glad to see this has been upgraded to numpy 1.18.  Any plans for a pypi release in the near future that would include this change?